src/constraints/constraints.jl

```diff
@@ -154,6 +154,16 @@ function conic_form!(c::GtConstraint, unique_conic_forms::UniqueConicForms)
     return get_conic_form(unique_conic_forms, c)
 end
 
+function add_constraints_to_context(lt::GtConstraint, context)
```

Comment:
I've only implemented `GtConstraint` so far, but the idea is to have such a method for each `Convex.Constraint` (like `conic_form!`). I did not use `template` because I think constraints should not return an objective function value but instead simply update the model (by adding constraints).